txclib/commands.py

```diff
@@ -50,13 +49,12 @@ def cmd_init(argv, path_to_tx):
 
     if os.path.isdir(os.path.join(path_to_tx, ".tx")):
         logger.info("tx: There is already a tx folder!")
-        reinit = input("Do you want to delete it and "
-                       "reinit the project? [y/N]: ")
-        while (reinit != 'y' and reinit != 'Y' and reinit != 'N'
-               and reinit != 'n' and reinit != ''):
+        reinit = False
+        neys = ['N', 'n', 'NO', 'no', 'No']
+        while (reinit not in ['y', 'Y', ''] + neys):
```

Comment:
Why not `while reinit.lower().startswith('y') or reinit.lower().startswith('n') or reinit == '':`